src/styles/components/_subscribe.scss

```diff
@@ -0,0 +1,79 @@
+@import '../settings/colors';
+@import '../tools/breakpoints';
+
+.w-subscribe {
+  display: block;
+  margin-top: 32px;
```

Comment:
Let's remove this and have the newsletter page define the spacing / margining for all of this child elements. Might be a good time to start introducing [the lobotomized owl selector](https://alistapart.com/article/axiomatic-css-and-lobotomized-owls/).